app/design/frontend/base/default/template/magmodules/webwinkelconnect/popup/privacy-popup.phtml

```diff
@@ -0,0 +1,10 @@
+<?php
+if (!$this->isPopupEnabled()) {
+    return;
+}
+
+echo sprintf(
+    '<script type="application/json" id="%s_order_completed">%s</script>',
+    htmlentities('webwinkelkeur'),
+    json_encode($this->getOrderData(), JSON_HEX_QUOT | JSON_HEX_TAG | JSON_HEX_AMP | JSON_HEX_APOS)
```

Comment:
You can add `JSON_INVALID_UTF8_SUBSTITUTE` to prevent invalid UTF-8 in one of the values from breaking the JSON entirely.